src/NServiceBus.Msmq.AcceptanceTests/When_distributing_a_command.cs

```diff
@@ -11,103 +11,178 @@
 
     public class When_distributing_a_command : NServiceBusAcceptanceTest
```

Comment:
that's related to the bug in the previous roundrobin distribution strategy which didn't track the index for each receiver properly. By using two different logical endpoints, this bug could be reproduced.